eslint.config.js

```diff
@@ -23,11 +23,7 @@ export default [
         files: ["tools/**"],
         rules: {
             "no-console": "off",
-            "n/no-process-exit": "off",
-            "n/no-unsupported-features/es-syntax": ["error", {
-                version: ">=16.0.0",
-                ignores: ["modules"]
-            }]
```

Comment:
This override is apparently no longer need with the latest version of eslint-plugin-n.